### PR TITLE
CITY-6: Added first post for Ideas in Progress

### PR DESCRIPTION
**Pull request by grrrero**
(Old PR [here](https://github.com/inclusive-design/website-cities/pull/3))

### CODE REVIEW

File: ideas/for-an-inclusive-quayside/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Co-designing Inclusive Cities - Ideas for an Inclusive Quayside</title>
+    <meta charset="utf-8">
+
+    <mCo-Designing eta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (colinbdclark) [inline ideas/for-an-inclusive-quayside/index.html]: Looks like there's a typo here, making your markup invalid. I think this should be a meta tag?

File: css/style.css [hunk 2]
```diff
@@ -128,6 +126,99 @@ a {
...
+	color: #e7e7e7;
+	margin: 0.8em 0;
+	font-size: 1.2em;
+	line-height: 1.6em;
+	letter-spacing: 0.04em;
+}
+
+/* Timeline
+	Brady Wright https://codepen.io/brady_wright/pen/NNOvrW
+	Niels Voogt https://codepen.io/NielsVoogt/pen/MbMMxv/
```
Comment (simonbates) [inline css/style.css]: How much content from others are we using? Do we know about licensing?

Comment (colinbdclark) [reply css/style.css]: From a note in the channel: all [source code posted in public CodePens are licensed under the MIT license](https://blog.codepen.io/legal/licensing/).

File: css/timeline.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
+/* Timeline
+    Brady Wright https://codepen.io/brady_wright/pen/NNOvrW
+    Copyright (c) 2018 by Brady Wright (https://codepen.io/brady_wright/pen/NNOvrW)
```
Comment (simonbates) [inline css/timeline.css]: I think we should reproduce the full license here.

File: ideas/for-an-inclusive-quayside/index.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+                <p>While Sidewalk Labs have announced their partnership with OCAD University to help with accessible design, the process is still in the early stages. The University's administration signed a letter of intent with Sidewalk that expresses its interest in collaborating. Here's what it says:</p>
+
+                <blockquote>&ldquo;We are excited at the prospect of working with OCAD University. To allow us the time to evaluate a possible collaboration and, if such a collaboration is feasible, come back to you with a developed scope and terms for a possible initial project, we are suggesting entering into this letter of intent.
+
```
Comment (colinbdclark) [inline ideas/for-an-inclusive-quayside/index.html]: I noticed that these line breaks don't appear when the text is rendered in the browser. It appears as single paragraph.

Comment (grrrero) [reply ideas/for-an-inclusive-quayside/index.html]: What's the best practice for this instead of hardcoding line breaks?

Comment (grrrero) [reply ideas/for-an-inclusive-quayside/index.html]: Altho this might change when we convert to markdown also...

Comment (simonbates) [reply ideas/for-an-inclusive-quayside/index.html]: We can put `<p>` paragraphs inside the `<blockquote>`:

https://www.w3.org/TR/html5/grouping-content.html#the-blockquote-element

`<p>` are "flow content"

Comment (colinbdclark) [reply ideas/for-an-inclusive-quayside/index.html]: Paragraph tags, I believe. Should be just like any other block-level container.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote#Example

Comment (colinbdclark) [reply ideas/for-an-inclusive-quayside/index.html]: ah, @simonbates beat me to it! :smile:

Comment (grrrero) [reply ideas/for-an-inclusive-quayside/index.html]: @colinbdclark @simonbates Made the changes

**Pull request merged**